pkg/cloudprovider/providers/gce/cloud/mock/mock.go

```diff
@@ -408,3 +408,28 @@ func UpdateRegionBackendServiceHook(ctx context.Context, key *meta.Key, obj *ga.
 	m.Objects[*key] = &cloud.MockRegionBackendServicesObj{Obj: obj}
 	return nil
 }
+
+// InsertFirewallsUnauthorizedErrHook mocks firewall insertion. A forbidden error will be thrown as return.
+func InsertFirewallsUnauthorizedErrHook(ctx context.Context, key *meta.Key, obj *ga.Firewall, m *cloud.MockFirewalls) (bool, error) {
+	return true, &googleapi.Error{Code: http.StatusForbidden, Message: fmt.Sprintf("Access denied.")}
```

Comment:
I think you can remove `Message: ...` from these functions. The compute API can be expected to return a StatusForbidden HTTP code, but we can't be 100% sure the message is always something like "Access denied".

Reply:
Make sense. Superfluous messages removed.